packages/fxa-shared/test/db/models/auth/helpers.ts

```diff
@@ -55,13 +60,17 @@ export function randomEmail(account: AccountIsh, primary = true) {
   };
 }
 
-export function randomEmailBounce(email: string): BounceIsh {
+export function randomEmailBounce(
+  email: string,
+  withDiagnosticCode: boolean = false
```

Comment:
Ohh I see what you mean -- I set it as the default here because it's being used in the `fxa-admin-server` script that we use for manually faking bounces, so I was thinking of it more as a dev tool than a test helper, and setting the default to false allowed it to keep being used the way it had been. It is confusing though. Would it be clearer if I added randomization to the tests? Or should I drop this altogether?